source/code/plugin/in_kube_health.rb

```diff
@@ -270,56 +271,56 @@ def process_node_condition_monitor(node_inventory)
       monitor_config = @provider.get_config(monitor_id)
       node_condition_monitor_records = []
       if !node_inventory.nil?
-          node_inventory['items'].each do |node|
-            node_name = node['metadata']['name']
-            conditions = node['status']['conditions']
-            node_state = HealthMonitorUtils.get_node_state_from_node_conditions(monitor_config, conditions)
-            details = {}
-            conditions.each do |condition|
-                condition_state = HealthMonitorStates::PASS
-                if condition['type'].downcase != 'ready'
-                    if (condition['status'].downcase == 'true' || condition['status'].downcase == 'unknown')
-                        condition_state = HealthMonitorStates::FAIL
-                    end
-                else #Condition == READY
-                    if condition['status'].downcase != 'true'
-                        condition_state = HealthMonitorStates::FAIL
-                    end
-                end
-                details[condition['type']] = {"Reason" => condition['reason'], "Message" => condition['message'], "State" => condition_state}
+        node_inventory["items"].each do |node|
+          node_name = node["metadata"]["name"]
```

Comment:
why this change? was this automatically done by VS code?

Reply:
yeah its the formatting tool i have. You can ignore those. It formats every file i open.